test/check-machines-create

```diff
@@ -287,14 +287,14 @@ class TestMachinesCreate(VirtualMachinesCase):
         b.enter_page('/machines')
         b.wait_in_text("body", "Virtual machines")
 
-        # First create the PXE VM but do not start it. We 'll need to tweak a bit the XML
```

Comment:
I see that this comment is after a few years of writing detaching from it's original code, but it's still referring to the XML editing done a few lines bellow. Line 317 and after.

I don't encourage keeping it, just clarifying that's actually still relevant.

Reply:
Right, but the "do not start it" is outright wrong. That's why I moved the comment about the XML fudging a little further down.